<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-RFNVP02" ignored="3" name="Default test(failed)" tests="2" failures="2" timestamp="07 Jan 2022 09:03:41 GMT" time="0.118" errors="0">
  <testcase name="tc5" time="0.027" classname="org.test.Sample">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.test.Sample.tc5(Sample.java:29)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- tc5 -->
  <testcase name="tc3" time="0.004" classname="org.test.Sample">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.test.Sample.tc3(Sample.java:17)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- tc3 -->
  <testcase name="tc2" time="0.0" classname="org.test.Sample">
    <ignored/>
  </testcase> <!-- tc2 -->
  <testcase name="tc1" time="0.0" classname="org.test.Sample">
    <ignored/>
  </testcase> <!-- tc1 -->
  <testcase name="tc4" time="0.0" classname="org.test.Sample">
    <ignored/>
  </testcase> <!-- tc4 -->
</testsuite> <!-- Default test(failed) -->
